src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -377,6 +377,28 @@ namespace OpenMS
       }
     }
 
+    // for fast pyOpenMS access to MS2 peak data in format: [rt, mz, intensity]
+    void getMS2PeakData(
```

Comment:
a level parameter could make sense here as we export all peaks (as opposed to the MS1 only area iterator version above)